packages/quick_actions/quick_actions/lib/quick_actions.dart

```diff
@@ -11,6 +11,9 @@ export 'package:quick_actions_platform_interface/types/types.dart';
 
 /// Quick actions plugin.
 class QuickActions {
+  /// const constructor for [QuickActions].
```

Comment:
Please describe what is happening when calling the constructor instead of having a verbose version of the code statement. Als make sure you capitalise the first character of your comment as per [Effective Dart's documentation section](https://dart.dev/guides/language/effective-dart/documentation#do-format-comments-like-sentences).

In this case I would suggest something like:
```dart
/// Creates a new instance of [QuickActions].
```